common/views/components/ToolbarSegmentedControl/README.md

```diff
@@ -0,0 +1,2 @@
+## Purpose
+
```

Comment:
relatable

Reply:
Ha. Possibly a good place for @GarethOrmerod to start his markdown training.